jaxws-ri/runtime/rt/src/main/java/com/sun/xml/ws/transport/http/client/HttpTransportPipe.java

```diff
@@ -372,19 +392,7 @@ private void recordCookies(Packet context, HttpClientTransport con) throws IOExc
         }
     }
 
-    private void addBasicAuth(Packet context, Map<String, List<String>> reqHeaders) {
-        String user = (String) context.invocationProperties.get(BindingProvider.USERNAME_PROPERTY);
-        if (user != null) {
-            String pw = (String) context.invocationProperties.get(BindingProvider.PASSWORD_PROPERTY);
-            if (pw != null) {
-                StringBuilder buf = new StringBuilder(user);
-                buf.append(":");
-                buf.append(pw);
-                String creds = DatatypeConverter.printBase64Binary(buf.toString().getBytes());
-                reqHeaders.put("Authorization", Collections.singletonList("Basic "+creds));
```

Comment:
this line is important as the http request must contain appropriate header - a test failure I see on Tomcat is very likely caused by this. I'll fix it later today